### PR TITLE
Fixes the visor of the welding mask going down when you use internals

### LINKED ISSUE #83728: Activating internals causes welder gas masks to lower
Reporting client version: 515.1639

## Round ID:
[229999](https://scrubby.melonmesa.com/round/229999)

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Borgs: the moduling (Various fixups to borg module code)](https://www.github.com/tgstation/tgstation/pull/83116)
- [Wawastation](https://www.github.com/tgstation/tgstation/pull/82298)
- [Incoming stamina damage while in stamcrit has diminishing returns applied](https://www.github.com/tgstation/tgstation/pull/83445)
- [Genetics Rebalance: Negative mutations add stability, standarized instability cost for mutations](https://www.github.com/tgstation/tgstation/pull/83439)
- [Cult Vs. Heretic: 7 Months Later Edition](https://www.github.com/tgstation/tgstation/pull/82877)

## Reproduction:
Strangely if you try to activate your internals with a welding gas mask on it'll activate both the visor and the internals rather than just the internals.

### PR DESCRIPTION
**Pull request by combustor**
## About The Pull Request
The visor of the welding mask no longer goes down when you enable internals

## Why It's Good For The Game
You'll no longer go semi-blind when you activate internals while using this mask
Fixes #83728 

## Changelog
:cl: Guestify
fix: The visor of the welding mask no longer goes down when you enable internals
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
this is a snowflake fix. you need to make it generic - apply it to all masks that don't need to be adjusted to have their internals work

**Comment by combustor** [comment]
Would this be okay? I didn't find any other mask that requires this exemption

File: code/_globalvars/bitfields.dm [hunk 1]
```diff
@@ -99,6 +99,7 @@ DEFINE_BITFIELD(clothing_flags, list(
 	"THICKMATERIAL" = THICKMATERIAL,
 	"VOICEBOX_DISABLED" = VOICEBOX_DISABLED,
 	"VOICEBOX_TOGGLABLE" = VOICEBOX_TOGGLABLE,
+	"INTERNALS_ADJUST_EXEMPT" =INTERNALS_ADJUST_EXEMPT,
```
Comment (san7890) [inline code/_globalvars/bitfields.dm:102]:
```suggestion
	"INTERNALS_ADJUST_EXEMPT" = INTERNALS_ADJUST_EXEMPT,
```

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -221,6 +221,7 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	visor_flags_cover = MASKCOVERSEYES
 	visor_vars_to_toggle = VISOR_FLASHPROTECT | VISOR_TINT
 	resistance_flags = FIRE_PROOF
+	clothing_flags = MASKINTERNALS | INTERNALS_ADJUST_EXEMPT
```
Comment (san7890) [inline code/modules/clothing/masks/gasmask.dm:224]:
```suggestion
	clothing_flags = parent_type::clothing_flags | INTERNALS_ADJUST_EXEMPT
```

this is to ensure we don't desynce from any changes on parent types.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Pull request merged**